WireAnalytics/Sources/WireAnalytics/Events/Segmentation/Predefined/Segmentation+Removed.swift

```diff
@@ -0,0 +1,57 @@
+//
+// Wire
+// Copyright (C) 2024 Wire Swiss GmbH
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+import struct Foundation.TimeInterval
+
+// The segmentation entries in this file are not to be used anymore.
+// As soon as we get the confirmation that these are not used anymore, we should delete this file.
+// https://wearezeta.atlassian.net/wiki/spaces/ENGINEERIN/pages/1364262933/Appendix+Countly+segmentation+values
+
+extension AnalyticsEvent.Segmentation {
+
+    enum Removed {
```

Comment:
I guess "removed" means these are to be deleted?

Reply:
I think so. I named it as it is named in Confluence:
https://wearezeta.atlassian.net/wiki/spaces/ENGINEERIN/pages/1364262933/Appendix+Countly+segmentation+values